src/main/java/com/paypal/heapdumptool/sanitizer/SanitizeOrCaptureCommandBase.java

```diff
@@ -47,15 +47,29 @@ public abstract class SanitizeOrCaptureCommandBase implements CliCommand {
     // match, all strings would be displayed as "*"
     private boolean forceMatchStringCoder;
 
-    @Option(names = {"-s", "--sanitize-byte-char-arrays-only"}, description = "Sanitize byte/char arrays only", defaultValue = "true", showDefaultValue = ALWAYS)
+    @Option(names = {"-s", "--sanitize-byte-char-arrays-only"},
+            description = "Sanitize byte/char arrays only",
+            defaultValue = "true",
+            arity = "1",
+            showDefaultValue = ALWAYS)
     private boolean sanitizeByteCharArraysOnly = true;
```

Comment:
change boolean fields to require arg (arity=1) because it's confusing otherwise especially for boolean fields that default to true.